requirements.txt

```diff
@@ -8,3 +8,4 @@ socketIO-client == 0.5.6
 stopit >= 1.1.1, < 1.1.999
 yamlconf >= 0.0.3, < 0.0.999
 revscoring >= 0.7.10
+redis
```

Comment:
Redis is an optional dependency.  You can run with an in-memory LRU (pylru) or no cache at all.